src/inner_op/somerc.rs

```diff
@@ -0,0 +1,298 @@
+#![allow(non_snake_case)]
+
+/// Swiss Oblique Mercator Projection
```

Comment:
It is fine to have docstrings in the code, but since the `inner_op`s live below the API surface, they are not public, and hence do not appear in the [docs](https://docs.rs/geodesy/latest/geodesy/).

This is obviously unfortunate, and may even be considered an error.

Currently, however, the inline documentation should be reflected in [Rumination #2](https://github.com/busstoptaktik/geodesy/blob/main/ruminations/002-rumination.md).

(Don't forget to add yourself to the author list at the top, and to update the "Document History" at the bottom)

Reply:
Ah yes, I totally forgot about the Rumination. I'll move this there.